docs/_includes/projects/docs/lifecycle/first/my-element.js

```diff
@@ -0,0 +1,14 @@
+import { LitElement, html } from '@polymer/lit-element';
+
+class MyElement extends LitElement {  
+  render(){
+    return html`
+      <textarea>Focus me on first update</textarea>
+    `;
+  }
+  firstUpdated(changedProperties){
+    console.log(changedProperties);
```

Comment:
Why are you doing `console.log(changedProperties)` here? It seems unnecessary.

Reply:
Sometimes I like to do console output to illustrate things to the reader. It's just a sample and they can ignore it if they want.